x/concentrated-liquidity/incentives.go

```diff
@@ -930,6 +930,18 @@ func (k Keeper) claimAllIncentivesForPosition(ctx sdk.Context, positionId uint64
 
 	supportedUptimes := types.SupportedUptimes
 
+	// Determine the qualifying balancer pool shares for the pools uptime accumulators
+	balancerPoolId, qualifyingBalancerShares, err := k.prepareBalancerPoolAsFullRange(ctx, position.PoolId, uptimeAccumulators)
+	if err != nil {
+		return sdk.Coins{}, sdk.DecCoins{}, err
+	}
```

Comment:
Note to reviewer: This is primary issue 1, not preparing balancer pool as full range prior to running claim/forfeit logic